.github/workflows/go.yml

```diff
@@ -0,0 +1,63 @@
+name: Go
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+
+  build:
+    strategy:
+      matrix:
+        go-version: [1.17, 1.18, 1.19]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/setup-go@v3
+      with:
+        go-version: ${{ matrix.go-version }}
+
+    - uses: actions/checkout@v3
+
+    - name: Build
+      run: go build -v ./...
```

Comment:
we could use `mage -v build` here, since the #74 is merged

Reply:
Done